curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-question-j.md

```diff
@@ -16,7 +16,7 @@ Adding `flex: 1` to `.item` makes each of the items grow to fill the available s
 
 Remove `flex: 1` from `.item` and add `justify-content: space-between` to `.container`. Doing so should give you something like this:
 
-<img src="https://cdn.statically.io/gh/TheOdinProject/curriculum/495704c6eb6bf33bc927534f231533a82b27b2ac/html_css/v2/foundations/flexbox/imgs/07.png" alt="an image displaying three blue squares which are spread out over the entire width of it's container">
+<img src="https://cdn.freecodecamp.org/curriculum/odin-project/flex-box/flexbox-05.png" alt="an image displaying three blue squares which are spread out over the entire width of it's container">
```

Comment:
This alt text is pretty good. Again, remove the "an image displaying" at the beginning. And I think we could give a little more detailed description about how these are spaced out, since the image is being used as an example of `justify-content: space-between`. Perhaps something like:

"Three small equally sized boxes representing flex items within a much larger rectangle representing a flex container. The small boxes are lined up in a single row, one touching the left edge of the container, one touching the right edge of the container, and the last directly in the middle of the container, placing as much space as possible between each box."

You might be able to shorten it up a little. The important part is to explain how these are spaced.